homeassistant/helpers/entity.py

```diff
@@ -1087,6 +1094,8 @@ def __async_calculate_state(
         available = self.available  # only call self.available once per update cycle
         state = self._stringify_state(available)
         if available:
+            if hasattr(self, "_attr_included_entities"):
+                attr[ATTR_ENTITY_ID] = self._attr_included_entities.copy()
```

Comment:
```suggestion
            if self.included_entities is not None:
                attr[ATTR_ENTITY_ID] = self.included_entities.copy()
```
We should also move it to `capability_attr`.

Reply:
I will have use for it in the follow-up PR. So maybe for now just rename `_attr_included_entities` to `included_entities` and initialise it to `None`.